client/elements/sc-donate-now-page.html

```diff
@@ -485,6 +484,16 @@
               return data;
           }
 
+          resetForm() {
+              this.$.donation_form.reset();
+              this.shadowRoot.querySelector('.amount-input').value = undefined;
+              this.shadowRoot.querySelector('.one-time-donation').checked = true;
+              this.shadowRoot.querySelector('.monthly-donation').checked = false;
+              this.shadowRoot.querySelector('.name-input').value = undefined;
+              this.shadowRoot.querySelector('.email-input').value = undefined;
+              this.shadowRoot.querySelector('.message-input').value = undefined;
```

Comment:
Are all these values not set by calling `this.$.donation_form.reset();`?

Reply:
No, that only resets the fields for the amount and the credit card details but not the rest.